swap_meet/item.py

```diff
@@ -1,2 +1,35 @@
+DESCRIPTION_DICT = {
+    0: "Did you even check?",
+    1: "Possibly good for parts, possibly.",
+    2: "A good holiday gift for your least favorite relative. ",
+    3: "Not bad...",
+    4: "Worth the price of admission.",
+    5: "A hidden treasure for Antiques Roadshow.",
+}
+
+
 class Item:
-    pass
\ No newline at end of file
+
+    def __init__(self, category = "", condition=0):
+        """
+        sets up the Item instance with an optional category attribute
+        """
+        
+        self.category = category
+        self.condition = condition 
+
+    def __str__(self):
+        return "Hello World!"
+
+    def condition_description(self):
+        """
+        Return a string description based on self.condition.
+        """
+
+        if self.condition < 0 or self.condition > 5:
```

Comment:
Nice! This covers any unwanted integer values but if `condition` is a string, you might end up with an unexpected result. You could try a ternary here to capture any condition that falls outside the dictionary keys. Also, note you could have used a tuple with an index (or even a list, although a tuple is probably a better choice).